### PR TITLE
rename BearerTokenFromServiceAccountToken

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
This PR:
* Renames BearerAccountFromServiceAccountToken to BearerAccountFromServiceAccount

### CODE REVIEW

File: api/observability/v1/clusterlogforwarder_types.go [hunk 1]
```diff
@@ -217,15 +217,15 @@ type BearerToken struct {
...
 type BearerTokenFrom string
 
 const (
 	// BearerTokenFromSecret specifies to use the token from the spec'd secret
 	BearerTokenFromSecret BearerTokenFrom = "secret"
 
-	// BearerTokenFromServiceAccountToken specifies to use the projected token associated with the forwarder service account
-	BearerTokenFromServiceAccountToken BearerTokenFrom = "serviceAccountToken"
+	// BearerTokenFromServiceAccount specifies to use the projected token associated with the forwarder service account
+	BearerTokenFromServiceAccount BearerTokenFrom = "serviceAccountToken"
```
Comment (vparfonov) [inline api/observability/v1/clusterlogforwarder_types.go:228]: "serviceAccountToken" -> "serviceAccount"?

Comment (jcantrill) [reply api/observability/v1/clusterlogforwarder_types.go]: good catch

**Pull request merged**